Dockerfile

```
@@ -13,8 +13,8 @@ RUN groupadd --system appuser \
     && useradd -g appuser -s /sbin/nologin -c "Docker image user" appuser
 
 WORKDIR /app
-COPY $jar app.jar
+COPY $jar blockchain-adapter-api.jar
```

We usually use the following format: `iexec-<app-name>.jar` ⬇
```suggestion
COPY $jar iexec-blockchain-adapter-api.jar
```